Automattic/wp-calypso
# Add: Backup Retention Disclaimer Information

#### Proposed Changes

Add a disclaimer to the bottom of Jetpack Backup Product cards that links to a FAQ on the pricing page that further describes the disclaimer.

When the Product Card is already on the pricing page, the link should jump down to the FAQ and open it up for the user.
When the Product Card is being used elsewhere, it should open a new window to https://cloud.jetpack.com/pricing and jump down to the FAQ and open it for the user.

P2 context: pbuNQi-3b3-p2

#### Testing Instructions

1. Check out this branch via `git switch add/backup-retention-disclaimer`
2. Run `yarn start` to start Calypso Blue
3. Navigate to http://calypso.localhost:3000/jetpack/connect/store and scroll until you can fully see the Jetpack backup card. There should be a disclaimer and link on the bottom
![Screen Shot 2022-07-22 at 4 22 51 PM](https://user-images.githubusercontent.com/65001528/180576207-481f88dc-b73f-46ba-8376-3ed895b1ee77.jpg)
4. Click on "Learn More." This should jump you down to the FAQ question and have it open when you get there
![Screen Shot 2022-07-22 at 4 24 14 PM](https://user-images.githubusercontent.com/65001528/180576287-5a567d04-a9af-4657-9b45-c454a90d3f01.jpg)
5. If you open and close the FAQ several times, you should see the hash value in the URL being removed when you close it, and added when you open it. This is to make it more convenient for users to link to specific questions, as well as make it easier to open up the FAQ again if the link is clicked a second time and the user had closed the FAQ from the first time.
6. Stop the local process and run `yarn start-jetpack-cloud`. Then navigate to http://jetpack.cloud.localhost:3000/pricing and repeat steps 3-5. Everything should work exactly the same as in Jetpack Connect
7. Navigate to http://jetpack.cloud.localhost:3000/pricing/storage/{ your-site }?calypso_env=development. From here you should see the same disclaimer on both of the cards that show up
![Screen Shot 2022-07-22 at 4 34 59 PM](https://user-images.githubusercontent.com/65001528/180577043-bc8e041d-5e25-4af5-8b6a-f503412fb1b2.jpg)
8. Click on this link, and a new window to http://jetpack.cloud.localhost:3000/pricing/ should pop up and take you directly to that FAQ question opened up just as before. Make sure you have `?calypso_env=development`, otherwise it will just take you to the live site
9. (optional) if you know of any other places these backup product cards are used on Clayp

#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
